shared/tasks/assets.mjs

```diff
@@ -14,15 +14,27 @@ export async function write (filePath, result) {
   const writeTasks = []
 
   // Files to write
+  /** @type {SourceMap | undefined} */
+  const map = result.map ? JSON.parse(result.map.toString()) : undefined
   const code = 'css' in result ? result.css : result.code
-  const map = result.map?.toString()
 
   // 1. Write code (example.js)
   writeTasks.push(writeFile(filePath, code))
 
   // 2. Write source map (example.js.map)
-  if (map) {
-    writeTasks.push(writeFile(`${filePath}.map`, map))
+  if (map && 'sources' in map) {
+    map.sources = map.sources
+
+      /**
+       * Make source file:// paths relative (e.g. for Dart Sass)
+       * {@link https://sass-lang.com/documentation/js-api/interfaces/CompileResult#sourceMap}
+       */
+      .map((path) => path.startsWith('file://')
```

Comment:
PostCSS adds the Dart Sass entry point `../src/all.scss` as a relative path

But all the other Dart Sass sources are absolute path `file://` URLs